h/api/views.py

```diff
@@ -157,8 +162,28 @@ def search(request):
 
 @api_config(route_name='access_token')
 def access_token(request):
-    """The OAuth 2 access token view."""
-    return request.create_token_response()
+    """Return a JWT access token for the given request.
+
+    The access token can be used in the Authorization of subsequent API
+    requests, to authenticate the request.authenticated_userid of the current
+    request.
+
+    """
+    try:
+        session.check_csrf_token(request, token='assertion')
+    except exceptions.BadCSRFToken:
+        raise httpexceptions.HTTPUnauthorized
+
+    expires_in = 3600
+    response = pyramid.response.Response(
```

Comment:
That is, if you just move this whole thing into `annotator_token` and get rid of `access_token`.